core/scss/components/molecules/_ctas.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+//
+// CTAs
+//
+// A call-to-action block with a clickable image and a button
+//
+// .decanter-cta--button-bottom           - Button under image link
+// .decanter-cta--button-bottom-icon      - Icon centered on image link with button underneath
+// .decanter-cta--button-center           - Button centered on image link
+//
+// Markup: ../templates/components/molecules/ctas/ctas.twig
+//
+// Style guide: Molecules.CTAs
+//
+
+.decanter-cta--button-bottom,
+.decanter-cta {
+  @include cta--button;
+}
+
+.decanter-cta--button-bottom-icon {
+  @include cta--button;
```

Comment:
As this is an additive css class, you won't need this mixin here.

Reply:
Removed.